app/config.py

```diff
@@ -77,5 +80,11 @@ class DevelopmentConfig(BaseConfig):
 
 
 class StageConfig(DevelopmentConfig):
+
+    required_config = ['S3_BUCKET_NAME', 'JWT_SEED',
```

Comment:
`API_KEY` (renamed to `JWT_SEED`) supposed to be supplied from external file or environment variable for staging and prod configuration.